pycfmodel/model/resources/properties/tag.py

```diff
@@ -1,5 +1,7 @@
+from typing import Optional
+
 from pycfmodel.model.resources.properties.property import Property
-from pycfmodel.model.types import ResolvableStr
+from pycfmodel.model.types import ResolvableBool, ResolvableStr
 
 
 class Tag(Property):
```

Comment:
Shall we create a child class adding this new property? This property is just for `AWS::AutoScaling::LaunchConfiguration` and doesn't apply to all other places.
(Is this detected in another object?)

Reply:
Like the idea! See [4b81f7f](https://github.com/Skyscanner/pycfmodel/pull/92/commits/4b81f7f8f215cd969db636ed85f54084eed2615e) for the changes.
With this change, we will need to update some internal rules as well.